packages/api/src/hooks/useGetAccountStatus.ts

```diff
@@ -0,0 +1,27 @@
+import { useMemo } from 'react';
+import useFetch from '../useFetch';
+
+/** A hook that retrieves the account status   */
+const useGetAccountStatus = () => {
+    const { data: get_account_status_data, ...rest } = useFetch('get_account_status');
+
+    // Add additional information to the authorize response.
+    const modified_account_status = useMemo(() => {
+        if (!get_account_status_data?.get_account_status) return undefined;
+
+        return {
+            ...get_account_status_data.get_account_status,
+            prompt_client_to_authenticate: Boolean(
+                get_account_status_data.get_account_status.prompt_client_to_authenticate
+            ),
```

Comment:
```suggestion
            /** Indicates whether the client should be prompted to authenticate their account. */
            should_prompt_client_to_authenticate: Boolean(get_account_status.prompt_client_to_authenticate),
```
@adrienne-deriv Since we are transforming it to a boolean it's better to also rename it to something more meaningful based on the data type and also add TSDoc.